libs/velo-external-db-core/src/router.ts

```diff
@@ -101,19 +100,23 @@ export const createRouter = () => {
         res.send(appInfoPage)
     })
 
-    router.get('/capabilities', async(req, res) => {
+    router.post('/capabilities/get', async(req, res) => {
         const capabilitiesResponse = {
-            capabilities: {
-                collection: [capabilities.CollectionCapability.CREATE]
-            } as capabilities.Capabilities
-        } as capabilities.GetCapabilitiesResponse
+            supportsCollectionModifications: true,
+            supportedFieldTypes: ['TEXT', 'NUMBER', 'DATE', 'DATETIME', 'BOOLEAN'],
+            supportsCollectionDisplayName: false,
+            supportsCollectionDisplayField: false,
+            supportsCollectionPermissions: false,
+            supportsCollectionFieldDisplayName: false,
+            supportsCollectionFieldDescription: false,
+        }
 
```

Comment:
Changed for the adapter to currently work with v3

will be further refined in the follow-up PR